src/components/notFoundBlock/notFoundBlock.jsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { PageMetadata } from "@docusaurus/theme-common";
+import Layout from '@theme/Layout';
+import './notFoundBlock.css';
+
+export const NotFoundBlock = () => {
+  return (
+      <>
+        <PageMetadata
+            title='Page Not Found'
+        />
```

Comment:
Please fix style for this file look like you have different indents

Reply:
I've checked it and it was OK as I remember, also I shown it to UX, but I'll recheck thanks

Reply:
I've used px instead of rem, may be it was a litile bit confusing